Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -247,24 +237,18 @@ function Invoke-TransformWebConfigAccountInitialization {
         $Config
     )
     Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $webConfigPath = "$($Config.PackageDirectory)/Web.config"
-
-        # Transform the web.config file using the initialization config
-        $transformParams = @{
-            sourceFile = "$($Config.PackageDirectory)/Web.config"
-            transformFile = "$PSScriptRoot/initialization.config"
-            destinationFile = $webConfigPath
-            toolsPath = $Config.ToolsPath
+        $webConfigPath = "$($Config.PackageDirectory)/appsettings.json"
+        $settings = Get-Content $webConfigPath | ConvertFrom-Json | ConvertTo-Hashtable
+
+        $InitializationSetting = @{
+            "{ACCOUNT_EMAIL}"    = $Config.AccountEmail
+            "{ACCOUNT_PASSWORD}" = $Config.AccountSecret
+            "{POPULATED_SECRET}" = $Config.PopulatedSecret
+            "{MINIMAL_SECRET}"   = $Config.MinimalSecret
```

Comment:
these settings need to look like the User object that I was trying to show you as an example.

Reply:
Sure Travis

Reply:
Added user object